### PR TITLE
micro: L2_POOL_2D PR3-5

### PR DESCRIPTION
**Pull request by ddavis-2015**
PR steps 3 through 5 for the L2_POOL_2D operator as per Issue #47814

### CODE REVIEW

File: tensorflow/lite/micro/kernels/micro_ops.h [hunk 1]
```diff
@@ -39,6 +39,8 @@ TfLiteRegistration Register_DEPTHWISE_CONV_2D();
 TfLiteRegistration Register_ELU();
 TfLiteRegistration Register_EXP();
 TfLiteRegistration Register_EXPAND_DIMS();
+TfLiteRegistration Register_L2_POOL_2D();
+TfLiteRegistration Register_LEAKY_RELU();
```
Comment (petewarden) [inline tensorflow/lite/micro/kernels/micro_ops.h:43]: This looks like an unrelated fix for Leaky Relu? Could you add it to a separate PR?

Comment (ddavis-2015) [reply tensorflow/lite/micro/kernels/micro_ops.h]: There was a merge conflict resolution error.  Fixed now.

**Review by petewarden — APPROVED** [review]
